execution/gethexec/executionengine.go

```diff
@@ -276,6 +311,80 @@ func (s *ExecutionEngine) SequenceTransactions(header *arbostypes.L1IncomingMess
 	})
 }
 
+func (s *ExecutionEngine) SequenceTransactionsEspresso(header *arbostypes.L1IncomingMessageHeader, rawTxes []espresso.Bytes) (*types.Block, error) {
+	return s.sequencerWrapper(func() (*types.Block, error) {
+
+		// Create the message. This message includes all the raw transactions from
+		// Espresso Sequencer.
+		msg := messageFromEspresso(header, rawTxes)
+
+		// Deserialize the transactions and ignore the malformed transactions
+		txes := types.Transactions{}
+		for _, tx := range rawTxes {
+			var out types.Transaction
+			if err := json.Unmarshal(tx, &out); err != nil {
```

Comment:
It seems here we are assuming that the `tx` bytes represent a JSON object. But IIUC these are the bytes returned by `FetchTransactionsInBlock`, which are not JSON-encoded but rather these are the raw bytes that were submitted to the sequencer. So for Arbitrum transactions I think this would be the RLP encoding of an EVM transaction.